benches/index-bench.rs

```diff
@@ -0,0 +1,39 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use tantivy::schema::{INDEXED, STRING, TEXT};
+use tantivy::Index;
+
+const HDFS_LOGS: &str = include_str!("hdfs.json");
```

Comment:
this includes the file in the binary

Reply:
yes I know :) but that's the bench binary.